fontra2fontir/src/fontra.rs

```diff
@@ -351,6 +415,10 @@ mod tests {
         font_data
             .axes
             .iter()
+            .map(|a| match a {
+                FontraAxis::Continuous(a) => a,
+                FontraAxis::Discrete(a) => panic!("Unexpected discrete axis: {a:#?}"),
```

Comment:
I don't think it's worth fixing here but I think the Continuous/Discrete and multiple codepoints could have been two separate PRs.